google/cloud/bigtable/internal/bigtable_stub_factory_test.cc

```diff
@@ -303,15 +306,15 @@ using ::testing::IsEmpty;
 
 TEST_F(BigtableStubFactory, TracingEnabled) {
   auto span_catcher = testing_util::InstallSpanCatcher();
-  auto propagator = testing_util::InstallMockPropagator();
-  EXPECT_CALL(*propagator, Inject);
 
   MockFactory factory;
   EXPECT_CALL(factory, Call)
       .WillOnce([](std::shared_ptr<grpc::Channel> const&) {
         auto mock = std::make_shared<MockBigtableStub>();
-        EXPECT_CALL(*mock, MutateRow)
-            .WillOnce(Return(internal::AbortedError("fail")));
+        EXPECT_CALL(*mock, MutateRow).WillOnce([](auto& context, auto const&) {
+          ValidatePropagator(context);
```

Comment:
Do you need a similar test for `storage_stub_factory.cc`?  It is also hand-crafted.

Reply:
Yeah, we should have a test. I blame you: #11369

Reply:
I'll send a PR when I get around to it.